$NetBSD: patch-solenv_bin_ooinstall,v 1.1 2012/07/28 20:28:29 ftigeot Exp $

The DESTDIR environment variable is leaking from the pkgsrc framework
and is now used by the LibreOffice 3.6 installation scripts in a way
which causes the installation prefix to be repeated for all files.

Minimally patch its use out to stop the madness.

--- solenv/bin/ooinstall.orig	2012-07-17 12:02:12.000000000 +0000
+++ solenv/bin/ooinstall
@@ -50,8 +50,8 @@ for $arg (@ARGV) {
 	$help = 1;
     } else {
 		# Cwd::realpath does not work if the path does not exist
-		mkpath($ENV{DESTDIR} . $arg) unless -d $ENV{DESTDIR} . $arg;
-		$path = Cwd::realpath( $ENV{DESTDIR} . $arg );
+		mkpath($arg) unless -d $arg;
+		$path = Cwd::realpath($arg);
     }
 }
 
